Frontend/ui-customer/src/Flow.purs

```diff
@@ -461,6 +462,17 @@ updateCustomerVersion dbClientVersion dbBundleVersion = do
     else pure unit
   else pure unit
 
+makeAppInstallsCall :: String -> FlowBT String Unit
+makeAppInstallsCall dummy = do
+  let versionName = getValueToLocalStore VERSION_NAME
```

Comment:
VersionName BundleVersion can be passed from baseAppFlow, instead of fetching it again from the storage. Then no need to pass a dummy string as well